packages/xrpl/src/Wallet/rfc1751Words.json

```diff
@@ -0,0 +1,243 @@
+[ "A", "ABE", "ACE", "ACT", "AD", "ADA", "ADD",
```

Comment:
It annoys me that these lines aren't even but that's really not worth fixing, it would be a pain